site/content/en/docs/Guides/metrics.md

```diff
@@ -119,6 +119,17 @@ helm install --wait --name prom stable/prometheus --namespace metrics \
 or `make kind-setup-prometheus` and `make minikube-setup-prometheus` for {{< ghlink href="/build/README.md#running-a-test-kind-cluster" branch="master" >}}Kind{{< /ghlink >}}
 and {{< ghlink href="/build/README.md#running-a-test-minikube-cluster" branch="master" >}}Minikube{{< /ghlink >}}.
 
+For resiliency it is recommended to run Prometheus on a dedicated node which is separate from nodes where Game Servers are scheduled. If you use `make setup-prometheus` to set up Prometheus, it will schedule Prometheus pods on nodes tainted with `stable.agones.dev/agones-metrics=true:NoExecute` and labeled with `stable.agones.dev/agones-metrics=true` if available.
```

Comment:
@kuqd remind me what's our standard here, for stating how we want to do this? Point to the yaml in GitHub?

Not sure if this should be a `feature` shortcode here to hide this until 0.8.0?

Reply:
can you update line 113 with the new command using the yaml and here just explain the same way that with a link to your yaml config.

Reply:
new command being 
```
helm upgrade --install --wait prom stable/prometheus --namespace metrics \
		--set alertmanager.enabled=false,pushgateway.enabled=false \
		--set kubeStateMetrics.enabled=false,nodeExporter.enabled=false \
		--set pushgateway.enabled=false \ 
--set server.global.scrape_interval=30s,server.persistentVolume.enabled=true,server.persistentVolume.size=64Gi  -f ./build/prometheus.yaml
```

Reply:
Awesome. I think once this is there, this is good for approval :+1:

Reply:
done